lib/resources/shim-head.js

```diff
@@ -2,6 +2,25 @@
 ((win) => {
   win.ELECTRON = true;
 
+  if (process.env.ELECTRON_PROTOCOL_SERVE_INDEX && window.location.protocol !== 'file:') {
```

Comment:
Note that this also allows the use case that the user has *two* ember apps bundled up in their electron app, and is using two different custom protocol handlers to serve them.